### DISCUSSION TITLE
How to create multi-page pdfs with different layouts

### DISCUSSION THREAD

**Discussion by seblic-ueda** [pdfme/pdfme#270] [Q&A]
Is it feasible? Also, if possible, ask how to do it.

I want to create a PDF with different layouts for page 1 and page 2.
How can I create a template file to make this possible?

**Comment by peteward** [comment]
Hi @seblic-ueda, 

pdfme does support multi-page PDFs - just upload one and start editing.

You can create separate schema variables for each page if you need to.

Or have I misunderstood your question?

**Reply by bgrand-ch** [reply]
Hello, thanks for answer.

How to use multiple `basePdf` in one template?

I would like page 1 to have a `basePdf`, page 2 to X another `basePdf` and the last page yet another `basePdf`.
